### PR TITLE
Change index page

### PR DESCRIPTION
**Pull request by Christen97**
Now the user is automatically redirected to the dashboard if they are logged in. If they are not logged in they get the outlook log in page.
The "guest page"(old index) is not used anymore.

### CODE REVIEW

**Comment by jazerix** [comment]
It shouldn't replace the current index page, just redirect to `/dashboard` if you are authenticated 😄

**Comment by Christen97** [comment]
> It shouldn't replace the current index page, just redirect to `/dashboard` if you are authenticated 😄

This is fixed now

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -16,8 +19,41 @@
 
 class DashboardController extends Controller
 {
-    public function index() : View
+    public function index(): View
     {
+        if (\Auth::user() == null)
+        {
```
Comment (jazerix) [inline app/Http/Controllers/DashboardController.php]: Everything within your if statement should be extracted to a private method within the controller. Also, use the `Auth::check()` method to ensure a user is logged in 😄

File: routes/web/web.php [hunk 2]
```diff
@@ -15,14 +15,11 @@
...
 use Illuminate\Support\Facades\Route;
-
-Route::get('/', [HomeController::class, 'index'])->name('home');
-
 Route::get('start', [HomeController::class, 'start'])->middleware('auth')->name('start');
 
 Route::get('status', [HomeController::class, 'status'])->middleware('auth')->name('status');
 
-Route::get('dashboard', [DashboardController::class, 'index'])->name('dashboard')->middleware('auth');
+Route::get('/', [DashboardController::class, 'index'])->name('home');
```
Comment (jazerix) [inline routes/web/web.php]: Can you move this back to the top of the file, while you're at it?

**Review by jazerix — CHANGES_REQUESTED** [review]
Almost there!

File: app/Http/Controllers/DashboardController.php [hunk 3]
```diff
@@ -16,27 +20,63 @@
...
+            'buildCount'      => $builds,
+            'buildAvg'        => $buildAverageTime,
+            'projectCount'    => $projectCount,
+            'assignmentCount' => $assignmentCount,
         ]);
     }
+
+    public function index(): view
+    {
+        if (Auth::check())
```
Comment (jazerix) [inline app/Http/Controllers/DashboardController.php:78]: Use a guard clause and we're there :D

**Review by jazerix — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/DashboardController.php (hunk 3 above)
Comment (jazerix) [reply app/Http/Controllers/DashboardController.php:78]: well not guard clause, but extract it to a method and return it instead

Comment (Christen97) [reply app/Http/Controllers/DashboardController.php:78]: I have extracted both methods now

**Review by jazerix — APPROVED** [review]

**Pull request merged**